orca-front50/src/main/groovy/com/netflix/spinnaker/orca/front50/pipeline/PipelineExpressionFunctionProvider.java

```diff
@@ -66,7 +75,7 @@ public Functions getFunctions() {
    *
    * @param execution the current execution
    * @param pipelineName name of the pipeline to lookup
-   * @return the id of the pipeline or null if pipeline not found
+   * @return the id of the pipeline or exception if pipeline not found
```

Comment:
@marchello2000 Ironically the documentation said it returned `null` here

Reply:
🤦‍♂ thank you!